Tools/build/stable_abi.py

```diff
@@ -654,7 +647,7 @@ def main():
     parser.add_argument(
         "--generate-all", action='store_true',
         help="as --generate, but generate all file(s) using default filenames."
-            + " (unlike --all, does not run any extra checks)",
+        " (unlike --all, does not run any extra checks)",
```

Comment:
Is it possible to align the strings here?

Reply:
Yes indeed